modules/osd-create-cluster-ccs.adoc

```diff
@@ -198,6 +201,26 @@ You must ensure that your VPC is configured with a public and a private subnet f
 ====
 endif::osd-on-aws[]
 
+. If you opted to configure a cluster-wide proxy, provide your proxy configuration details on the *Cluster-wide proxy* page:
++
+--
+.. Complete at least one of the following fields:
+** Specify a valid *HTTP proxy URL*.
+** Specify a valid *HTTPS proxy URL*.
+** Optional: In the *No proxy domains* field, specify any domains or Classless Inter-Domain Routing (CIDR) ranges that should not use the proxy. To exclude all subdomains for a domain, you can use `\*` as a wildcard. For example, you can specify `*.example.com` to to prevent all traffic for the example.com domain and its subdomains from using the proxy.
```

Comment:
The no_proxy is not supported on UI page currenlty

Reply:
This docs PR is dependent on https://issues.redhat.com/browse/HAC-698 and https://issues.redhat.com/browse/HAC-710 and will be merged after the feature is promoted.

Reply:
The **No proxy domains** UI step has now been removed from the PR.